translations/admin+intl-icu.en.xlf

```diff
@@ -341,6 +341,14 @@
         <source>Select receipt template</source>
         <target>Select receipt template</target>
       </trans-unit>
+      <trans-unit id="MtlQbTd" resname="List of custom fields(one per line). Should not contain any spaces.">
```

Comment:
A space is missing before “(one per line)”.

Reply:
Fixed.